src/transactionsFactories/tokenManagementTransactionsFactory.ts

```diff
@@ -402,6 +418,10 @@ export class TokenManagementTransactionsFactory {
         attributes: Uint8Array;
         uris: string[];
     }): Transaction {
+        if (!options.uris.length) {
+            throw new ErrBadUsage("");
```

Comment:
Breaking change? Or fixing change? Message is missing.

Reply:
this is now deleted.